.spelling

```diff
@@ -107,6 +107,7 @@ Kube
 Kubecon
 Kubelet
 Kubernetes
+kubernetes
```

Comment:
The initial of `Kubernetes` should be capitalized. No need the add this spelling.